packages/battery/LICENSE

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Your Company. All rights reserved.
```

Comment:
Good catch, updated.

It's impossible to know which places need updating after running `flutter create --plugin` :(